src/elements/link-list/common.ts

```diff
@@ -0,0 +1,3 @@
+export * from './common/link-list-base.js';
```

Comment:
Is this extra file really needed? could it have the link-list in it's file name?

Reply:
I see it's the same pattern used in `src/elements/button/common.ts`

Reply:
ok :)